### PR TITLE
Update branch strategy in docs

### PR DESCRIPTION
**Pull request by aletss**
- Add initial documentation for git strategy including branching and PR guidelines

### CODE REVIEW

File: docs/git_strategy.md
Comment (Quiroptero) [inline docs/git_strategy.md]: We have a little typo here: it should say "further".

Also, it's a matter of taste and style, but what do you think of using [semantic line breaks](https://sembr.org/)? In this case it would break this line in two lines:
```markdown
The pull request should have a descriptive title.
It should describe the changes, any further action needed and if possible specify the testing.
```

**Comment by Quiroptero** [comment]
Question: what do you think of specifying both `reviewers` and `assignees` for all pull requests?

**Comment by aletss** [comment]
> Question: what do you think of specifying both `reviewers` and `assignees` for all pull requests?

I'm not used to it but I like the idea.

**Review by Quiroptero — APPROVED** [review]
LGTM

**Comment by aletss** [comment]
What do you think if the owner of the PR is the one that has to merge it after the approval?

**Comment by Quiroptero** [comment]
> What do you think if the owner of the PR is the one that has to merge it after the approval?

Sure 😅, I'm not used to it but it's ok for me

**Pull request merged**
